Classes/Event/EventBus.php

```diff
@@ -1,54 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\Cqrs\Event\Exception\EventBusException;
-use Neos\Cqrs\EventListener\EventListenerLocator;
-use TYPO3\Flow\Annotations as Flow;
-use TYPO3\Flow\Log\SystemLoggerInterface;
-use TYPO3\Flow\Utility\TypeHandling;
-
-/**
- * @Flow\Scope("singleton")
- */
-class EventBus
```

Comment:
What does stripping the EventBus has to do with allowing metadata enrichment?

And the EventBus is the central layer of abstraction for the event transport. If you want to support a distributed system, this is the single infrastructure point that has to be replaced in order to move events from one server to another. I'm really not sure why this should be removed at all. If too much functionality was moved to the EventPublisher, we should rather consider moving that functionality back.